### DISCUSSION TITLE
Hello Blobby Boi can u further upgrade ext hanger?

### DISCUSSION THREAD

**Discussion by ghost** [3kh0/ext-remover#1326] [Fixes JOIN TITANIUM NETWORK TO SHARE PLEASE]
I cant use it and it just works for like 20 mins and then it just keep on showing me save changes and then when I click prevent pop ups it just goguardian just blocks the site I was on. When I press cancel it just shows me more pop ups and then it gets annoying. Pls Help

**Comment by S-PScripts** [comment]
say it on his repo

**Reply by ghost** [reply]
thanks
